site/content/integrate/apps/deployment/_index.md

```diff
@@ -52,40 +49,128 @@ For a go app the manifest snippet would look like this:
         }
     ]
 }
-``` 
+```
+
+### Setting up your Mattermost instance and AWS for deploying custom apps
+
+Note that custom apps are not supported in the Mattermost Cloud - they're only used for developer testing convenience. Provisioning in the AWS cloud environment is done by the **appsctl** tool after completing the environment set up.
```

Comment:
What does `environment set up` refer to? Can we link something?

Reply:
```suggestion
Note that custom apps are not supported in the Mattermost Cloud - they're only used for developer testing convenience. Provisioning in the AWS cloud environment will be done by the **appsctl** tool in the AWS setup documentation section
```

Reply:
I changed the wording a bit to preface that the tool *will be* used in a later section.

Reply:
Also removes the `environment set up` language.  It was referring to the compilation of the AWS setup and setting the ENV Vars.

Reply:
See [`32e4489` (#826)](https://github.com/mattermost/mattermost-developer-documentation/pull/826/commits/32e4489e837bfeb88d2e0e599aebc22a438813ee)